package.json

```diff
@@ -8,21 +8,24 @@
         "url": "https://github.com/gosling-lang/gosling.js"
     },
     "homepage": "https://gosling-lang.github.io/gosling.js/",
-    "main": "dist/gosling.js",
+    "main": "dist/gosling.es.js",
```

Comment:
"main" applies to both ES module and CJS module entry points. There is no `dist/gosling.js` created anymore so we point this to `gosling.es.js`